c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
I wonder if this change is correct. OK for testing first whether the document has a geometry attribute or not (eg. articles). But are we sure ``doc_v1['geometry']`` is never ``None`` when there's a ``geometry`` attribute?
@tsauerwein what do you think?

In addition I think the indentation could be improve. It would be more readable to write 
```
'geometry1': doc_v1['geometry']['geom']
    if 'geometry' in doc_v1 else None,
```
wouldn't it?